### DISCUSSION TITLE
F86AC158-FD5E-409E-83A1-6FEB3120F4F8

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#62] [General]
# Earth: History (Archean and Hadean Eons)

An addon that brings a spectacular view of the history of our planet Earth, from 4.5 billion to 20 million years in the past. This addon adds 46 alternate surfaces for Earth, showing what our planet might have looked like at different times.

https://celestia.mobi/addon?item=F86AC158-FD5E-409E-83A1-6FEB3120F4F8&lang=zh_CN

**Comment by 13940951572** [comment]
怎么下载不了

**Comment by Minecraft633** [comment]
In my opinion this is better than Paleomap

**Comment by vinotintocorp-create** [comment]
Hey Comments, We need to Fix It.
The Animated Earth's Birth (4.5 BYA) to work harder to make $10M.
Animated 1,000,000,000 Asteroids Colliding the planet (No Glitching) $9.5M
Animated Hot Burning Fire entering atmosphere over $9M
Animated Theia Collision to Earth is Animated over $9.7M.
